web/app/components/categoriesList/categories.tsx

```diff
@@ -0,0 +1,46 @@
+import { Flex, Anchor } from "@mantine/core"
+
+export function Categories() {
+  return (
+    <>
```

Comment:
I think we can remove this wrapping fragment since the `Flex` is already wrapping the components